### PR TITLE
Require dotenv-webpack

### PR DESCRIPTION
**Pull request by bermudezmt**
**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [ ] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

**Comment by vernig** [comment]
Hey @bermudezmt let me know if you need help with this PR...

File: webpack.config.js [hunk 1]
```diff
@@ -1,8 +1,27 @@
...
-   * Consult https://webpack.js.org/configuration for more information
-   */
-
-  return config;
+    /**
+     * Customize the webpack by modifying the config object.
+     * Consult https://webpack.js.org/configuration for more information
+     */
+    // We dynamically change the path to the .env that contains the file corresponding to our profile
+    let envPath;
```
Comment (ktalebian) [inline webpack.config.js:8]: This is now automatic - provided the name of the profile matches the envPath, i.e. `.env.${process.env.TWILIO_PROFILE}` would automatically be injected

**Comment by bermudezmt** [comment]
Hey @vernig , please feel free to take over / incorporate this PR into #29 which I'll review and test tomorrow. Thanks again for all the help!

File: webpack.config.js (hunk 1 above)
Comment (bermudezmt) [reply webpack.config.js:8]: As this is already automatic, maybe we can just close this PR?

**Comment by bermudezmt** [comment]
I'm guessing we can close this one now as well? Feel free to do so, @vernig .

**Comment by vernig** [comment]
Closing this. The officially supported method for having multiple _staging_ `.env` files is to create a file named `.{profile-id}.env` where `{profile-id}` is the ID (name) of the profile selected on the Twilio CLI (you can check this name using `twilio profiles:list`)

**Pull request closed without merging**